pkg/renderer/sgml/blank_line.go

```diff
@@ -0,0 +1,29 @@
+package sgml
+
+import (
+	"bytes"
+	"github.com/bytesparadise/libasciidoc/pkg/types"
+
+	log "github.com/sirupsen/logrus"
+)
+
+func (sr *sgmlRenderer) renderBlankLine(ctx *Context, _ types.BlankLine) ([]byte, error) {
```

Comment:
I know it's a tiny thing, but could we use single letter for receiver names? Here, I believe that `r` (for "renderer") is enough. That's something I've tried to do in the rest of the code, so I'd like to keep things consistent here.

Reply:
Funny, but I started out that way (it was originally t, because instead of sgmlRenderer it was templates, before I renamed the package.)

The problem was that as I was converting a lot of existing functions to move them into this case, there were a lot of name collisions.  That may or may not be the case with "r" or "s" on their own right now -- I haven't checked.  But "sr" is sufficiently unique that there are no collisions whatsoever.

I can try to do a rename of this mechanically with goland and see what happens.  If there are no collisions, I'm not adverse.

Reply:
ok, thanks!

Reply:
So both "s" and "r" have conflicts.  (I tested both via mechanical renaming.)  I'd suggest to just let this one go.  We already have ctx anyway.

Reply:
ah too bad. I will probably give it a try then, in a more "manual" fashion. Are you ok if I make a PR on your PR so you can see what I want to change, rather than pushing on your PR directly?

Reply:
Sure, that's fine.